wallet/errors.go

```diff
@@ -7,6 +7,8 @@ import (
 
 var (
 	ErrIsolatedWalletCantGenerateKeyPairs = errors.New("isolated wallet can't generate key pairs")
+	ErrIsolatedWalletCantGetMasterKeyPair = errors.New("isolated wallet can't get master key pair")
```

Comment:
I think "isolated wallet don't have a master key" is more appropriate. @wwestgarth thoughts ?

Reply:
Agree, I'd go with "isolated wallet doesn't have a master key"